prov/gni/test/rdm_atomic.c

```diff
@@ -481,7 +484,7 @@ void do_min(int len)
 
 	cr_assert_eq(ret, 1, "fi_cq_read returned %d %d", ret,
 			dump_cq_error(send_cq[0], target, 0));
-	rdm_atomic_check_tcqe(&cqe, target, FI_ATOMIC | FI_WRITE, 0);
+	rdm_atomic_check_tcqe(&cqe, target, FI_ATOMIC | FI_WRITE, 0, ep[1]);
```

Comment:
Actually, looking more closely in the GNI provider, completion events are not generated on the remote peer for atomic operations, only counters are used. I can't tell from the man page, is this intended? If so, we can simply reset rdm_atomic.c. Sorry for the confusion and thank you for catching this mistake.